doc-content/enterprise-only/openshift/proc-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-dashbuilder-standalone_{context}"]
+= Installing Dashbuilder Standalone using Operator
+
+You can use Operator to install Dashbuilder Standalone independently from other services.
+
+.Procedure
+
+. On the *Installation* page, in the *Application name* field, enter `standalone-dashbuilder`.
```

Comment:
Minor suggested rewrite:

On the *Installation* page, enter `standalone-dashbuilder` in the *Application name* field.

Reply:
I agree that this reads better. I based my phrasing on the "Field" examples found at IBM SG 209p.